src/Compatibility/Compatibility.sln

```diff
@@ -0,0 +1,50 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
We don't need to check in the solution if we don't want to but I found it easier and seems like everytime that somebody works on PackageValidation will have to touch ApiCompat or viceversa.

Reply:
It seems like the sdk pattern is to instead add an `slnf` rather than a new solution. The Validation projects are already part of the sdk.sln.

Reply:
oh good call, I forgot about the slnf, I'll remove the sln then